src/Cache/Internal/ScsDataClient.php

```diff
@@ -2031,6 +2038,75 @@ function () use ($call, $value): SortedSetGetScoreResponse {
         );
     }
 
+    /**
+     * @return ResponseFuture<SortedSetUnionStoreResponse>
+     */
+    public function sortedSetUnionStore(
+        string $cacheName,
+        string $destination,
+        array $sources,
+        ?int $aggregate = AggregateFunction::SUM,
+        ?CollectionTtl $ttl = null
+    ): ResponseFuture
+    {
+        try {
+            $collectionTtl = $this->returnCollectionTtl($ttl);
+            // The number of source sets is currently limited to 2. I'm just adding this validation and
+            // some validation of the contents of $sources here for now, but will move to _DataValidation
+            // eventually.
+            if (count($sources) > 2) {
+                throw new InvalidArgumentError("The number of source sets is currently limited to 2");
+            }
```

Comment:
Are we leaving this at <= 2 for this PR or did you intend to remove this?

Reply:
This is an explicit service limit, so I was planning to keep this code and comment around until we can know the best way to validate this more formally.